Initialtask/src/Entity/Product.php

```diff
@@ -0,0 +1,167 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\ProductRepository;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Table(name="tblProductData")
+ * @ORM\Entity(repositoryClass=ProductRepository::class)
+ */
+class Product
```

Comment:
It's hard to maintain stability in Anaemic models, please remove setter or make them private(setters can be used for business logic validation as well if this business logic belongs to this domain model) and add single point of truth:
```
private function __constructor($your_arguments) {
...
} 
public static function create($your_argumenta) {
  return new self(...);
}
```